webapp/app/perf.html

```diff
@@ -36,6 +36,9 @@
   <script src="vendor/mousetrap.min.js"></script>
   <script src="js/treeherder.js"></script>
   <script src="js/providers.js"></script>
+  <script src="js/models/option_collection.js"></script>
+  <script src="js/services/main.js"></script>
+  <script src="js/services/log.js"></script>
```

Comment:
OptionCollection model uses  thUrl service  which is in main.js  and thUrl service is using  ThLog, it is in log.js. They are the services being used by option collection model.